.jenkins/Jenkinsfile

```diff
@@ -0,0 +1,77 @@
+#!/usr/bin/groovy
+@Library('jenkins-pipeline@v0.4.5')
+import com.invoca.docker.*;
+
+pipeline {
+  agent {
+    kubernetes {
+      defaultContainer "ruby"
+      yamlFile ".jenkins/ruby_build_pod.yml"
+    }
+  }
+
+  environment {
+    GITHUB_TOKEN = credentials('github_token')
+    BUNDLE_GEM__FURY__IO = credentials('gemfury_deploy_token')
+    RAILS_ENV = 'test'
```

Comment:
I was surprised to not find this already here. @jebentier do you know if we have it set at a higher level?

Reply:
This should be set in the `spec_helper`.

Reply:
Ok, removed from here.